test/test_loss.py

```diff
@@ -0,0 +1,82 @@
+"""Unit tests for the loss function."""
+from __future__ import print_function
+import json
+import numpy as np
+import tensorflow as tf
+from wavenet import make_loss, compute_receptive_field_size
+
+
+def make_logits():
+    # 1 batch item
+    # 5 time steps in duration
+    # 3 channels (quantization levels)
+    array = np.array([[[5.0,   -10.0, -10.0],
+                       [10.0,  -10.0, -10.0],
+                       [0.0,    10.0,   0.0],
+                       [-10.0,  10.0, -10.0],
+                       [-10.0,  10.0, -20.0]]])
+
+    return tf.convert_to_tensor(array, tf.float32)
+
+
+def make_labels():
+    # 1 batch item
+    # 5 time steps in duration
+    # 3 channels (quantization levels)
+    array = np.array([[[0.0, 1.0, 0.0],
+                       [0.0, 0.0, 1.0],
+                       [0.0, 0.0, 1.0],
+                       [0.0, 0.0, 1.0],
+                       [0.0, 0.0, 1.0]]])
+    return tf.convert_to_tensor(array, tf.float32)
+
+
+class TestLoss(tf.test.TestCase):
```

Comment:
Define a class for each function that you test (make_loss and compute_receptive_field_size)